owncloudDomain/src/main/java/com/owncloud/android/domain/availableoffline/AvailableOfflineRepository.kt

```diff
@@ -0,0 +1,27 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Abel García de Prada
+ * Copyright (C) 2022 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.owncloud.android.domain.availableoffline
+
+import com.owncloud.android.domain.files.model.OCFile
+
+interface AvailableOfflineRepository {
+    fun setFileAsAvailableOffline(fileToSetAsAvailableOffline: OCFile)
+    fun unsetFileAsAvailableOffline(fileToSetAsAvailableOffline: OCFile)
```

Comment:
I don't know if it's suitable that both methods have a parameter with the same name which indicates a same aim, when their objective is precisely the opposite. Maybe changing the name of the second one to `fileToUnsetAsAvailableOffline`?